adapters/emx_digital/emx_digital.go

```diff
@@ -238,6 +286,19 @@ func (a *EmxDigitalAdapter) MakeBids(internalRequest *openrtb.BidRequest, extern
 
 }
 
+func getBidType(impID string, imps []openrtb.Imp) openrtb_ext.BidType {
+	bidType := openrtb_ext.BidTypeBanner
+	for _, imp := range imps {
+		if imp.ID == impID {
+			if imp.Video != nil {
+				bidType = openrtb_ext.BidTypeVideo
+			}
+			break
+		}
+	}
+	return bidType
+}
```

Comment:
Does your bid response include the bid type? Although the code you have here is common in the project, there's been issues reported (see https://github.com/prebid/prebid-server/issues/862) that this approach isn't always reliable for multiformat impressions. Is there enough information to detect the type based on the bid response?

Reply:
Thank you very much for the fix here.